flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionTest.java

```diff
@@ -282,61 +278,6 @@ public void testSlotAllocationCancellationWhenExecutionCancelled() throws Except
 		assertThat(canceledSlotRequests, equalTo(slotRequests));
 	}
 
-	/**
-	 * Tests that all preferred locations are calculated.
-	 */
-	@Test
-	public void testAllPreferredLocationCalculation() throws Exception {
-		final TaskManagerLocation taskManagerLocation1 = new LocalTaskManagerLocation();
-		final TaskManagerLocation taskManagerLocation2 = new LocalTaskManagerLocation();
-		final TaskManagerLocation taskManagerLocation3 = new LocalTaskManagerLocation();
-
-		final CompletableFuture<TaskManagerLocation> locationFuture1 = CompletableFuture.completedFuture(taskManagerLocation1);
-		final CompletableFuture<TaskManagerLocation> locationFuture2 = new CompletableFuture<>();
-		final CompletableFuture<TaskManagerLocation> locationFuture3 = new CompletableFuture<>();
-
-		final Execution execution = getExecution(Arrays.asList(locationFuture1, locationFuture2, locationFuture3));
-
-		CompletableFuture<Collection<TaskManagerLocation>> preferredLocationsFuture = execution.calculatePreferredLocations(LocationPreferenceConstraint.ALL);
-
-		assertFalse(preferredLocationsFuture.isDone());
-
-		locationFuture3.complete(taskManagerLocation3);
-
-		assertFalse(preferredLocationsFuture.isDone());
-
-		locationFuture2.complete(taskManagerLocation2);
-
-		assertTrue(preferredLocationsFuture.isDone());
-
-		final Collection<TaskManagerLocation> preferredLocations = preferredLocationsFuture.get();
-
-		assertThat(preferredLocations, containsInAnyOrder(taskManagerLocation1, taskManagerLocation2, taskManagerLocation3));
-	}
-
-	/**
-	 * Tests that any preferred locations are calculated.
-	 */
-	@Test
-	public void testAnyPreferredLocationCalculation() throws Exception {
-		final TaskManagerLocation taskManagerLocation1 = new LocalTaskManagerLocation();
-		final TaskManagerLocation taskManagerLocation3 = new LocalTaskManagerLocation();
-
-		final CompletableFuture<TaskManagerLocation> locationFuture1 = CompletableFuture.completedFuture(taskManagerLocation1);
-		final CompletableFuture<TaskManagerLocation> locationFuture2 = new CompletableFuture<>();
-		final CompletableFuture<TaskManagerLocation> locationFuture3 = CompletableFuture.completedFuture(taskManagerLocation3);
-
-		final Execution execution = getExecution(Arrays.asList(locationFuture1, locationFuture2, locationFuture3));
-
-		CompletableFuture<Collection<TaskManagerLocation>> preferredLocationsFuture = execution.calculatePreferredLocations(LocationPreferenceConstraint.ANY);
-
-		assertTrue(preferredLocationsFuture.isDone());
-
-		final Collection<TaskManagerLocation> preferredLocations = preferredLocationsFuture.get();
-
-		assertThat(preferredLocations, containsInAnyOrder(taskManagerLocation1, taskManagerLocation3));
-	}
-
```

Comment:
Just to confirm: We can remove these tests because we no longer use `LocationPreferenceConstraint.ANY` in the new scheduler, right?

Reply:
yes. The new scheduler uses `DefaultPreferredLocationsRetriever` and `LocationPreferenceConstraint ` is not used anymore.

Reply:
Thanks for the clarification @zhuzhurk.